deploy-all.sh

```diff
@@ -1 +1,5 @@
-./node_modules/.bin/concurrently "./deploy-backend.sh" "./deploy-frontend.sh"
\ No newline at end of file
+cd wellbeing/app && meteor npm i && cd ../../
+cd wellbeing-client/frontend && yarn && cd ../../
+
+echo "export PATH=$PATH:$(npm get prefix)/bin" >> ~/.bashrc
```

Comment:
Why do we need to modify .bashrc for this deployment? That seems like an unnecessary side-effect. Can't we use a session variable in the shell script?

Reply:
it is not able to find local dependencies